e2etest/bgp_test.go

```diff
@@ -150,6 +152,14 @@ var _ = ginkgo.Describe("BGP", func() {
 					Name:      "bgp-test",
 					Protocol:  BGP,
 					Addresses: poolAddresses,
+					BGPAdvertisements: []bgpAdvertisement{
+						{
+							Communities: []string{
```

Comment:
Will these be available to parse on the receiver side?
If so the tests should verify that.

Reply:
done w/o the need to snoop the packet on the receiver peer